common/flatpak-dir.c

```diff
@@ -16766,20 +16777,21 @@ flatpak_dir_list_unused_refs (FlatpakDir         *self,
 
               /* deploy v4 guarantees eol/eolr info */
               deploy_data = flatpak_dir_get_deploy_data (self, ref, 4,
-                                                         cancellable, NULL);
+                                                          cancellable, NULL);
               is_eol = deploy_data != NULL &&
                 (flatpak_deploy_data_get_eol (deploy_data) != NULL ||
-                 flatpak_deploy_data_get_eol_rebase (deploy_data));
+                  flatpak_deploy_data_get_eol_rebase (deploy_data));
             }
 
-          if (!is_eol)
-            {
-              g_debug ("%s: Ref %s (%s) not end-of-life, so excluding from EOL unused refs",
-                       G_STRFUNC, flatpak_decomposed_get_ref (ref),
-                       flatpak_dir_get_name_cached (self));
+          if (!((is_autopruned && filter_by_autoprune) || (is_eol && filter_by_eol)))
+          {
```

Comment:
Is the indentation correct here? (I genuinely don't know if GitHub is tricking me with this diff, but the file view shows the indentation off too, so I'll assume it's a valid review comment)

Reply:
My editor hated this indenting style :P